Makefile

```diff
@@ -116,6 +117,11 @@ test-e2e:
 .PHONY: test-gator
 test-gator: gator test-gator-verify test-gator-test test-gator-expand
 
+.PHONY: test-gator-containerized
+test-gator-containerized: __test-image
+	docker run --privileged -v $(shell pwd):/app -v /var/lib/docker \
```

Comment:
this is going to make artifacts created by this (like gator binary) owned by root. can we do this without privileged?

Reply:
The test needs to run docker-inside-docker in order to spin up a local container registry to push to test bundles to. Unfortunately, `--privileged` is necessary to allow the inner container (the container registry) to be able to access the file system of the outer container (the test runner). 

I'm not sure I understand why this would make the gator binary owned by root, as this is privileged execution is just for a test image. Is there someway we can exclude it? Otherwise, if this is a problem, we might have to explore other methods of e2e testing this, like perhaps using an external public registry for the test images instead of spinning up a local one.